docs/registers/ps.md

```diff
@@ -9,31 +9,35 @@ comments: true
 # Parallel Input Serial Output 
 {: .no_toc }
 
+## Table of contents
+{: .no_toc .text-delta }
 
+1. TOC
+{:toc}
 
 
 ## Introduction
  
-* Data bits are entered in parallel fashion.
+* Data bits are entered in a parallel fashion.
 
-* The circuit shown below is a four bit parallel input serial output register.
+* The circuit shown below is a four-bit parallel input serial output register.
 
 * Output of previous Flip Flop is connected to the input of the next one via a combinational circuit.
 
-* The binary input word B0, B1, B2, B3 is applied though the same combinational circuit.
+* The binary input word B0, B1, B2, B3 is applied through the same combinational circuit.
 
 * There are two modes in which this circuit can work namely - shift mode or load mode.
 
 ## Load mode
 When the shift/load bar line is low (0), the AND gate 2, 4 and 6 become active they will pass B1, B2, B3 bits to the corresponding flip-flops. 
-On the low going edge of clock, the binary input B0, B1, B2, B3 will get loaded into the corresponding flip-flops. 
+On the low going edge of the clock, the binary input B0, B1, B2, B3 will get loaded into the corresponding flip-flops. 
```

Comment:
content from http://www.brainheaters.in/uploads/2/6/0/3/26031536/adc.pdf

Reply:
I have made some changes to the docs. Please review them as well